### DISCUSSION TITLE
How can I get the sequence a line W in GFA is referencing?

### DISCUSSION THREAD

**Discussion by syresbr** [ComparativeGenomicsToolkit/cactus#1427] [Q&A]
Hi, I used the Cactus-Minigraph to generate a pan-genome of sugar-cane and I would like to get the sequence present in the line W in the GFA to understand the genes presents there. How can I use it?

Other thing, how can I see what parts of the pan-genome are core, acessory and exclusive? I made a script in python to see the number of segments present in each genome and by that define if the line W is core, acessory or exclusive, but I think there might be a better way to do it. 

Thanks for your time.

**Comment by glennhickey** [comment]
You can use `vg paths -F` to extract paths in the graph as FASTA sequences. 

You can use `vg depth` (maybe binning with `-b`) to get the number of paths covering each position in the graph, which may help with your second question.  Otherwise there may be something in `odgi`.  I agree we should ship easier scripts for this type of analysis.
